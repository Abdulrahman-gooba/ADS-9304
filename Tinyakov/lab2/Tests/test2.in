(cos(*(256)(sin(-(r)))))r
