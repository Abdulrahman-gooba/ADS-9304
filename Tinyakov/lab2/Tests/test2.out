-sin((256)*(sin(-(r))))*(((0)*(sin(-(r)))+(256)*(cos(-(r))*(-(1)))))
